apps/api/src/profiles/profiles.service.ts

```diff
@@ -102,7 +102,10 @@ export class ProfilesService {
   async updateUser(authId: string, updateProfileDto: UpdateProfileDto) {
     const EmailResp = await this.getUserByEmail(updateProfileDto.email);
     if (EmailResp.data != null) {
```

Comment:
Go for a `Boolean(userByEmail)` . Also `getUserByEmail`. Why would u accept undefined here. 

Follow a single casing system and a bit more good naming convention

Reply:
Merged for now